openslides/motions/models.py

```diff
@@ -35,6 +36,26 @@
 from .exceptions import WorkflowError
 
 
+class ConstitutionParagraph(RESTModelMixin, models.Model):
+    """
+    Model for parts of the constitution
+    """
+    access_permissions = ConstitutionParagraphAccessPermissions()
+
+    title = models.CharField(max_length=255)
+    """Title of the constitution paragraph."""
+
+    text = models.TextField()
+    """Content of the constitution paragraph."""
+
+    class Meta:
+        default_permissions = ()
+        ordering = ['title']
```

Comment:
It is not self-evident that we can order this by title. In some cases we may have an alpha-numeric title prefix but there may also other cases. So I think we need a weight field too.

Reply:
ok